scripts/cat/cats.py

```diff
@@ -3365,7 +3364,7 @@ def sprite(self, new_sprite):
     # ---------------------------------------------------------------------------- #
 
     def is_baby(self):
-        return self.age in ["kitten", "newborn"]
+        return self.age in [CatAgeEnum.KITTEN, CatAgeEnum.NEWBORN]
```

Comment:
Would like to suggest moving this check into the CatAgeEnum class, so it could be called by `cat.age.is_baby()`